src/Microsoft.Windows.CsWin32/Generator.TypeDef.cs

```diff
@@ -319,6 +338,41 @@ private IEnumerable<MemberDeclarationSyntax> CreateCommonTypeDefMembers(Identifi
             .WithSemicolonToken(SemicolonWithLineFeed);
     }
 
+    private List<string> GetAlsoUsableForValues(EntityHandle handle)
+    {
+        QualifiedTypeDefinitionHandle tdh;
+        if (handle.Kind == HandleKind.TypeReference)
+        {
+            if (!this.TryGetTypeDefHandle((TypeReferenceHandle)handle, out tdh))
+            {
+                throw new GenerationFailedException("Unable to look up type definition.");
+            }
+        }
+        else if (handle.Kind == HandleKind.TypeDefinition)
+        {
+            tdh = new QualifiedTypeDefinitionHandle(this, (TypeDefinitionHandle)handle);
+        }
+        else
+        {
+            throw new GenerationFailedException("Unexpected handle type.");
+        }
+
+        List<string> alsoUsableForValues = new();
+        QualifiedTypeDefinition td = tdh.Resolve();
+        foreach (CustomAttributeHandle ah in td.Definition.GetCustomAttributes())
+        {
```

Comment:
Actually since evidently what we want is the `TypeDefinition` and our caller already has it, we should just pass that in directly rather than pass in any handle at all and require this method to re-look it up.

Reply:
I'm not sure what the equivalent calls are later on if we pass in just a TypeDefinition. Could you fix this one?

Reply:
I found `this.Reader`. Is that the equivalent of `td.Reader`? It seems to work.